tello_sim/command_server.py

```
@@ -280,6 +280,36 @@ def _handle_connection(self, conn: socket.socket) -> None:
                 state = "unhealthy"
             conn.sendall(state.encode())
 
+        # --- scene editing (tools/scene_editor.py) ---------------------------
+        # These only ever queue an edit; the adapter applies it on the main
+        # thread next tick, because rebuilding a gate mesh from this socket
+        # thread would be Panda3D scene-graph surgery off the render thread.
+        elif data == "get_scene":
+            conn.sendall(json.dumps(self._ursina_adapter.scene_snapshot()).encode())
+        elif data.startswith("set_gate ") or data.startswith("set_person "):
```

Ruff (0.16.0)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)